src/3rd_party-static/jsoncpp/include/json/reader.h

```diff
@@ -135,9 +135,9 @@ namespace Json {
       bool readCppStyleComment();
       bool readString();
       void readNumber();
-      bool readValue();
-      bool readObject( Token &token );
-      bool readArray( Token &token );
+      bool readValue( int *const depth_level );
```

Comment:
@ZhdanovP why are you using pointers instead of refs here?

Reply:
@ZhdanovP depth_level is an output parameter isn't it? Please name it properly and create description for the function. I would also create interface without this parameter. In most cases I don't case what was deep of nested structures if I was not able to read value.